tests/core/recipes/compiled_recipes/test_real_orca_recipes.py

```diff
@@ -1,12 +1,18 @@
 from shutil import which
+import os
 
 import pytest
 from ase.build import molecule
 from numpy.testing import assert_allclose
 
 from quacc.recipes.orca.core import relax_job, static_job
 
-has_orca = bool(which("orca"))
+orca_path = which("orca")
+
+if orca_path and os.path.getsize(orca_path) > 1024 * 1024:
```

Comment:
@rishav-karanjit: Thanks!! This looks good to me.

One question: do you know the executable size of the Ubuntu orca screen reader? Is it indeed < 1 Mb?

Reply:
Yes. In fact, its in kb.